x-pack/legacy/plugins/infra/public/components/navigation/routed_tabs.tsx

```diff
@@ -54,6 +56,24 @@ export class RoutedTabs extends React.Component<RoutedTabsProps> {
   }
 }
 
+const tabBetaBadgeLabel = i18n.translate('xpack.infra.common.tabBetaBadgeLabel', {
+  defaultMessage: 'Beta',
+});
+
+const tabBetaBadgeTooltipContent = i18n.translate('xpack.infra.common.tabBetaBadgeTooltipContent', {
+  defaultMessage:
+    'This feature is under active development. Extra functionality is coming, and some functionality may change.',
+});
+
+export const TabBetaBadge = euiStyled(EuiBetaBadge).attrs({
```

Comment:
I factored this out in anticipation of the upcoming categorization tab.

Reply:
Worth a thought. Although I think for an "official" badged tab I feel it would require more polish. The position and weight don't really feel like an annotation to me.